### PR TITLE
add adminClaimStates and fix actual cost update bug

### PR DESCRIPTION
**Pull request by hobbitronics**
- I noticed the customer and admin see different banner while looking at the same claim
adminNeedsReviewClaim

### CODE REVIEW

File: src/data/states.ts [hunk 1]
```diff
@@ -73,18 +80,33 @@ export const claimStates: { [stateName: string]: State } = {
...
   Approved: { ...approved, title: 'Approved' },
   Inactive: { ...pending, title: 'This item has no coverage', icon: 'umbrella' },
   Pending: { ...pending, title: 'Awaiting item coverage review' },
 }
 
-export const getClaimState = (status: ClaimStatus): State => {
-  if (claimStates[status] === undefined) {
-    console.error('No such state (for claim status):', status, Object.keys(claimStates))
+export const getClaimState = (status: ClaimStatus, isAdmin = false): State => {
+  if (claimStates[status] === undefined && adminClaimStates[status] === undefined) {
```
Comment (forevermatt) [inline src/data/states.ts:102]: Doing the check here, before narrowing down to whether `isAdmin` is true or false, means we could ask for an admin state but pass `false` for isAdmin and see neither the console error nor the state we were expecting.

What do you think of doing an `assertHas()` here? It's not what we typically use them for... just a thought.

Feel free to change this or not, as seems best to you for now.

**Review by forevermatt — APPROVED** [review]
Approved, with a comment/question

**Comment by forevermatt** [comment]
Looks like the build failed due to a type error:
```
17:49:03.426 | src/pages/policies/[policyId]/claims/[claimId].svelte
17:49:03.428 | Error: [svelte-preprocess] Encountered type error
17:49:03.428 | at throwError (/opt/buildhome/repo/node_modules/svelte-preprocess/dist/modules/errors.js:5:11)
```

File: src/data/states.ts (hunk 1 above)
Comment (hobbitronics) [reply src/data/states.ts]: Good point. I would rather have better error handling than this.

**Pull request merged**